README.md

```diff
@@ -4,7 +4,7 @@
 
 > Project Microverse Trials Code Review
 
-![screenshot](./app_screenshot.png)
```

Comment:
Sorry but I don't understand your code review. This line have you appoint me doesn't have in the branch, only in master and I'll merge both just after your approval here.

Reply:
> Sorry but I don't understand your code review. This line have you appoint me doesn't have in the branch, only in master and I'll merge both just after your approval here.

HI @leonardoeng13, so sorry i made a mistake reviewing this, as it is a repeated project i recommend creating a new repo for it, so sorry 🙏